src/main/java/org/scijava/console/DefaultConsoleService.java

```diff
@@ -179,11 +195,23 @@ private void cacheListeners() {
 		cachedListeners = listeners.toArray(new OutputListener[listeners.size()]);
 	}
 
-	private MultiPrintStream multiPrintStream(final PrintStream ps) {
+	private static MultiPrintStream multiPrintStream(final PrintStream ps) {
 		if (ps instanceof MultiPrintStream) return (MultiPrintStream) ps;
 		return new MultiPrintStream(ps);
 	}
 
+	private static PrintStream skipMultiPrintStream(final PrintStream ps) {
+		if (! (ps instanceof MultiPrintStream) )
+			return ps;
+		return printStream(((MultiPrintStream) ps).getDestination());
```

Comment:
It's been awhile since I've studied or thought about this logic, but might this need to be recursive, in case of multiple layers of `MultiPrintStream` wrapping?